src/views/Strategy.vue

```diff
@@ -6,49 +6,50 @@
       </router-link>
     </div>
     <div>
-      <div class="col-12 col-lg-8 float-left pr-0 pr-lg-5">
```

Comment:
The strategy page is broken the content is not shown https://bafybeie7ui2lll252cv3tekjocx7zs2x7wuvujyiudpnwdsviutpgsrajm.on.fleek.co/#/strategy/erc20-balance-of-delegation